src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
Consider using `arguments.length` to check if no arguments are passed, instead of checking if `num` is `undefined`. This will ensure that the function behaves correctly even if `undefined` is explicitly passed as an argument.